RELEASE_NOTES.md

```diff
@@ -9,6 +9,8 @@
 - The `CanStreamData` RPC has been removed. Users are recommended to check it
   by calling the `StreamComponentData` RPC.
 
+- `frequenz-api-common` has been updated to `v0.3.1`.
```

Comment:
I guess you should also update the minimum requirement in the `pyproject.toml` file :grimacing: 

https://github.com/frequenz-floss/frequenz-api-microgrid/blob/v0.x.x/CONTRIBUTING.md#upgrading-dependencies

Reply:
Creating a new PR

Reply:
https://github.com/frequenz-floss/frequenz-api-microgrid/pull/100